photon-core/src/main/java/org/photonvision/common/dataflow/networktables/NTDataPublisher.java

```diff
@@ -61,6 +66,9 @@ public class NTDataPublisher implements CVPipelineResultConsumer {
     private final Supplier<Integer> pipelineIndexSupplier;
     private final BooleanSupplier driverModeSupplier;
 
+    private final PipelineManager pipelineManager =
+            new PipelineManager(new CameraConfiguration(null, null));
+
```

Comment:
Seems a bit like code leakage to maintain this state in two places? In both VisionModule and NTDataPublisher. Can the logging calls instead potentially move to the VisionModule itself? Or is there another way around this?